third_party/move/move-vm/runtime/src/storage/module_storage.rs

```diff
@@ -125,6 +127,27 @@ pub trait ModuleStorage: WithRuntimeEnvironment {
             .ok_or_else(|| module_linker_error!(address, module_name))
     }
 
+    /// Returns the module without verification, or [None] otherwise. The existing module can be
+    /// either in a cached state (it is then returned) or newly constructed. The error is returned
+    /// if the storage fails to fetch the deserialized module.
+    #[cfg(fuzzing)]
+    fn fetch_module(
```

Comment:
nit: can we call this `fetch_module_skip_verification`? So that it is clear from the name